docs/arcaflow/workflows/output.md

```diff
@@ -24,5 +24,5 @@ A common example of two mutually exclusive events could be the availability of y
 In this example taken from the [Arcaflow Workflows](https://github.com/arcalot/arcaflow-workflows/blob/main/example-workflow/workflow.yaml) project, the `success` output collects the data from the specified steps and inserts it into data storage. The `no-indexing` output collects the data, the error logs, and does not store the data.
 
 ```yaml
-{!https://raw.githubusercontent.com/arcalot/arcaflow-workflows/main/system-performance/sysbench/workflow.yaml [ln:90-100]!}
-```
\ No newline at end of file
+{!https://raw.githubusercontent.com/arcalot/arcaflow-workflows/88d07bf83c286eedae76fde72f5165b577e68927/advanced-examples/system-performance/sysbench/workflow.yaml [ln:102-112]!}
```

Comment:
👍🏻 

As much as I'd prefer not to do something like this, doing it against a particular commit sounds much better than a path off main. (Of course inevitably someone will get frustrated that it doesn't auto-update and end up breaking it again, but here we are...)

Reply:
Huh.  I was so happy to actually find the file and to get the reference working again that I didn't stop to consider the ramifications of that particular choice of URL.

As you say, referring to a particular commit does have the advantage that it is unlikely to break again...but, as you say, someone might conceivably be confused when the contents of the reference don't get updated...but, given that no one noticed when the reference was outright broken, I doubt anyone will be bothered if the contents don't change.